### PR TITLE
Update backend start to use docker

### PR DESCRIPTION
**Pull request by andrewpeng02**
Because the new training backend relies on starting `celery` and `redis`, I created a docker compose file to start those services in one command, and changed the `backend start` to run docker compose. See https://github.com/DSGT-DLP/Deep-Learning-Playground/pull/1183

https://github.com/DSGT-DLP/dlp-cli/assets/47485510/5fb711a0-bdd9-4355-8ca8-4cfbc547de09

### CODE REVIEW

**Comment by karkir0003** [comment]
@andrewpeng02 did you attach testing to show the functionality works?

File: cmd/backend/start/start.go [hunk 1]
```diff
@@ -14,15 +12,13 @@ import (
 var StartCmd = &cobra.Command{
 	Use:   "start",
 	Short: "Starts the training backend",
-	Long:  `Starts an instance of the training backend Django app in /training in the terminal`,
+	Long:  `Starts an instance of the training backend Django app in /training in the terminal. To change the backend port, set the environment var BACKEND_PORT. `,
```
Comment (karkir0003) [inline cmd/backend/start/start.go:15]: so would the invocation still be `dlp-cli backend start`?

**Review by karkir0003 — COMMENTED** [review]
left a quick question

File: cmd/backend/start/start.go (hunk 1 above)
Comment (andrewpeng02) [reply cmd/backend/start/start.go:15]: yes

**Review by karkir0003 — APPROVED** [review]

**Pull request merged**